pypeit/display/display.py

```diff
@@ -283,9 +284,8 @@ def show_points(viewer, ch, spec, spat, color='cyan', legend=None, legend_spec=N
     canvas.add('constructedcanvas', canvas_list)
 
 
-
 # TODO: Should we continue to allow rotate as an option?
-def show_slits(viewer, ch, left, right, slit_ids=None, left_ids=None, right_ids=None, maskdef_ids=None, spec_vals = None,
+def show_slits(viewer, ch, left, right, slit_ids=None, left_ids=None, right_ids=None, maskdef_ids=None, spec_vals=None,
```

Comment:
@debora-pe I modified show_slits and show_trace in the sensfunc_blaze_jwst_lists branch to also take 2d arrays of traces or slits. That will probably generate conflicts. No action item now, but I may need your help with the merge.

Reply:
Sounds good!